docs/guides/go-built-in.md

```diff
@@ -101,10 +101,18 @@ github.com/cometbft/cometbft v0.38.0
 )
 ```
 
+XXX: CometBFT `v0.38.0` uses a slightly outdated `gogoproto` library, which
+may fail to compile with newer Go versions. To avoid any compilation errors,
+upgrade `gogoproto` manually:
+
+```bash
+go get github.com/cosmos/gogoproto@v1.4.11
```

Comment:
Should we upgrade the gogoproto version we use on the `v0.38.x` branch?

Reply:
https://github.com/cometbft/cometbft/pull/1609